tests/testdata/run/rejection_handled.ts

```diff
@@ -14,4 +14,4 @@ setTimeout(async () => {
 
 setTimeout(() => {
   console.log("Success");
-}, 50);
+}, 200);
```

Comment:
@bartlomieju - I needed to change this to make this test pass on windows. otherwise `Success` is printed too early. perhaps code cache changed the execution timing a bit and 50ms wasn't enough.

Reply:
Is it possible it was a flake on Windows? I don't see anything in your PR that should affect this. This smells like a deno_core bug.

I'm also not sure why we are using an async function in the setTimeout above. That _could_ be part of the problem.

Reply:
It's possible that it flaked on Windows; it failed consistently for several runs though. I'll try to do more runs with/without this change.

Reply:
I confirmed again that the test is consistently failing on windows, and a higher timeout makes the output appear as expected.